stable/elasticsearch-exporter/README.md

```diff
@@ -44,6 +44,7 @@ The following table lists the configurable parameters of the Elasticsearch-Expor
 
 Parameter | Description | Default
 --- | --- | ---
+`appVersion` | Application Version | `1.0.2`
```

Comment:
no need to introduce a new parameter for the version since it's already defined in the `Chart.yaml`. You can access the value using `{{ $.Chart.AppVersion }}`

Reply:
I've tried to use `{{- if hasPrefix "1.1" $.Chart.AppVersion }}`, but it didn't work, so `appVersion` was introduced. As there is no need to support the older version, I will remove this variable